### ISSUE TITLE
Reverse output option?

### ISSUE THREAD

**Issue by Robin-Andrews** [chicolucio/truth-table-generator#1]
Great work on this.

Is there a way to have the propositions display in "ascending order"?
e.g.

0 0
0 1
1 0
1 1

for p, q.

**Comment by chicolucio** [comment]
Hi! Sorry to take so long to reply.

The order is due to these lines (152 and 153) in ttg.py file:

```
self.base_conditions = list(itertools.product([True, False],
                                                      repeat=len(bases)))
```

Changing the True/False order would generate your output. 

Maybe an option, let's call it `ascending`, could be implemented so that the user could change this order. Something like this:

```
print(ttg.Truths(['p', 'q'], ascending=True))
```

The default value would be False, so that people using the module the way it is now will not need to change their code nor see any difference to their current uses.

I don't have time right now, so if you or someone wants to contribute this functionality that would be great. But I will definitely take a look at this when possible.

**Comment by chicolucio** [comment]
Closed by PR #2
